documentation/content/content-nodes.html

```diff
@@ -213,94 +217,69 @@ <h3 id="master-election">Master election</h3>
+content nodes are polled for bucket handover -
+a distributor will then handle a new set of buckets.
 </p><p>
-The distribution algorithm also calculates what content nodes copies of buckets should be stored on.
-As moving bucket copies take time, the distributors track current state in a bucket database,
-allowing requests to be processed independent of where buckets are currently located.
-Distributors use the distribution algorithm to detect buckets that are stored on wrong nodes,
-and move copies to correct nodes when there are free resources to do so.
+Document API clients do not interface with the cluster controller,
+and does hence not know the cluster start at startup.
+A random distributor is used first. <!-- ToDo: Or does it use the ideal state algo? -->
+If the document operation hit the wrong distributor,
+<em>WRONG_DISTRIBUTION</em> is returned, with the current cluster state.
+<em>WRONG_DISTRIBUTION</em> is hence expected and normal at cold start / state change events.
 </p><p>
-In addition the distributor can track some historic knowledge.
-For instance it may know that of the currently existing bucket copies,
-a given copy has been available all the time and may be trusted to have all the information,
-while another copy may be on a storage node that recently restarted,
-so that copy may lack some documents.
-Such historic state is not persisted, and is thus lost on distributor restarts.
 </p><p>
-The distributors use the bucket metadata to ensure the buckets are in a good state.
+</p>
+
+
+<h3 id="maintenance-operations">Maintenance operations</h3>
+<p>
+Distributors track which content nodes have which buckets in their bucket batabase.
+Distributors then use the <a href="idealstate.html">ideal state algorithm</a>
+to generate bucket <em>maintenance operations</em>.
+A stable system has all buckets located per the ideal state:
 <ul>
-  <li>If buckets have too few copies, new copies are generated</li>
-  <li>If buckets have too many copies, superfluous copies are deleted once the
-    distributor knows the copies to delete don't contain data other copies do not have</li>
-  <li>If all the copies do not contain the same data,
-    merges are issued to get bucket copies consistent</li>
+    <li>If buckets have too few replicas, new are generated on other content nodes.</li>
+  <li>If the replicas differ, a bucket merge is issued to get replicas consistent.
+      The distributor sets a bucket replica <em>trusted</em>. <!-- ToDo: need something here - this is confusing - old text below:
+      
+      For instance it may know that of the currently existing replicas,
+      a given replica has been available all the time and may be trusted to have all the information,
+      while another replica may be on a content node that recently restarted,
+      so that replicas may lack some documents.
+      Such historic state is not persisted,
+      and is thus lost on distributor restarts.--></li>
```

Comment:
The "trusted" replica notion is iffy as it's a transient property and therefore is unreliable (ironically enough, given the name) during node up/down edges. I suggest we remove the sentence mentioning it entirely.